Adding unlimited-open-source section to docs #453 (#461)
gitpod-io/website: src/docs/unlimited-open-source.md

# Unlimited Open Source

It'd not have been possible for us to create Gitpod without all the amazing open-source work, it's built upon. And still, we're developing huge parts of Gitpod in open source. 

Deeply devoted to the open source community, we are happy to provide professional open-source contributors a free **Unlimited Open Source** plan. 

##What does it include?
No more hour restrictions, use Gitpod for unlimited hours on any public GitHub or GitLab repository.

##Who is eligible? 

> You should fulfill at least one of the following criteria:

Reviewer: I think the usual terms are "meet" ... "requirements":

```suggestion
You should meet at least one of the following requirements:
```

1. You are a maintainer, or core contributor, of a well-established open source project
2. A significant part of your revenue comes from open source work (e.g. via Liberapay, Open Collective, GitHub Sponsors, Patreon, etc)
3. You regularly contribute to open source in other ways (e.g. producing content like blog posts, videos, live streams, or organizing conferences, meet-ups, hackathons)

> Is at least one criterion applying to your work? Please, [get in contact](/contact/) and send us a link to your projects. We'd love to see what you're working on. 

Reviewer: Suggestion based on the previous one (also "get in touch" sounds more natural to me, and there may often be more than one link):

```suggestion
Does at least one of the above apply to your work? Then please [get in touch](/contact/) and provide links to your projects. We'd love to see what you're working on. 
```
